code/modules/shuttle/on_move.dm

```diff
@@ -396,10 +396,34 @@ All ShuttleMove procs go here
 
 	. = ..()
 
+/obj/docking_port/mobile/fob/onShuttleMove(turf/newT, turf/oldT, rotation, list/movement_force, move_dir, old_dock)
+	. = ..()
+
+	//Unload the planet
+	var/obj/docking_port/stationary/S = old_dock
+	for(var/datum/star_system/SYS in SSstarmap.star_systems)
+		var/datum/planet/PL = SYS.get_planet_for_z(oldT.z)
+		if(S.name == "FOB Landing Zone" && SSstarmap.current_planet.name != PL.name)
+			PL.no_unload_reason = ""
+			//PL.do_unload()
+	for(var/mob/living/simple_animal/M in range(30, S.loc)) //Stops the shuttle from landing on top of mobs
+		qdel(M)
```

Comment:
you don't need this, shuttles innately handle landing on mobs though I think I have some notes somewhere about needing to tweak it a bit

Reply:
your use of range() here hurts me

Reply:
They gib whatever they land on. In the case of necropolis tendrils, this will create chasms inside the shuttle.

Reply:
Also I *did* ping you on discord to see if there was a better way to do this.